docs/faq.md

```diff
@@ -187,34 +197,43 @@ All Syne Tune and user metadata are saved when the tuner starts under `metadata.
 
-If you run experiments with tabulated benchmarks using the `SimulatorBackend`, as demonstrated in https://github.com/awslabs/syne-tune/blob/main/examples/launch_nasbench201_simulated.py, results are stored by `SimulatorCallback` instead, and you need to inherit from this class, as shown in https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/searchers/searcher_callback.py#L88.
+If you run experiments with tabulated benchmarks using the `SimulatorBackend`, as demonstrated in 
+[launch_nasbench201_simulated.py](https://github.com/awslabs/syne-tune/blob/main/examples/launch_nasbench201_simulated.py), results are stored by `SimulatorCallback` instead, and you need to inherit from this class, as shown in 
+[searcher_callback.py](https://github.com/awslabs/syne-tune/blob/main/syne_tune/optimizer/schedulers/searchers/searcher_callback.py#L88).
 
 ### <a name="remote-tuning"></a> I don’t want to wait, how can I launch the tuning on a remote machine?
 
 You can use the remote launcher to launch a tuning on a remote machine. The remote launcher supports both `LocalBackend` and `SagemakerBackend`. In the former case, multiple trials will be evaluated on the remote machine (one use-case being to use a beefy machine), in the latter case trials will be evaluated as separate SageMaker training jobs.
 
-See for an example on how to run tuning with the remote launcher: https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_sagemaker_remotely.py
+See for an example on how to run tuning with the remote launcher: [launch_height_sagemaker_remotely.py](https://github.com/awslabs/syne-tune/blob/main/examples/launch_height_sagemaker_remotely.py)
 
 ### <a name="experiment-parallel"></a> How can I run many experiments in parallel?
 
-You can call the remote launcher multiple times to schedules a list of experiments. In some case, you will want more flexibility and directly write your experiment loop, you can check https://github.com/awslabs/syne-tune/tree/main/benchmarking/benchmark_loop for an example.
+You can call the remote launcher multiple times to schedules a list of experiments. In some case, you will want more flexibility and directly write your experiment loop, you can check 
+[benchmark_loop](https://github.com/awslabs/syne-tune/tree/main/benchmarking/benchmark_loop) for an example.
 
 ### <a name="results-remote-tuning"></a> How can I access results after tuning remotely?
 
 You can either call `load_experiment("{tuner-name}")` which will download files from s3 if the experiment is not found locally. You can also sync directly files from s3 under `~/syne-tune/` folder in batch for instance by running:
 
 ```
-aws s3 sync s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/ ~/syne-tune/
+aws s3 sync s3://{sagemaker-default-bucket}/syne-tune/{tuner-name}/ ~/syne-tune/  --include "*"  --exclude "*tuner.dill"
```

Comment:
This is fine. My example was rather doing --exclude "*", then include results.tar.gz and metadata.json. But this is fine as well, as long as the user sees how this is done.